Libraries/Test-Framework.psm1

```diff
@@ -540,18 +540,7 @@ function Run-Test {
              -Timeout $timeout
         $resultArr += $testResult
     }
-
-    $currentTestResult.TestResult = GetFinalResultHeader -resultarr $resultArr
```

Comment:
why do you change the order here

Reply:
@mbivolan - please run PAUSED-CRITICAL-HEARTBEAT TC to see if these changes are affecting that TC

Reply:
The clenup script must be executed before the VM is removed.